kzg10-rust/kzg-bench/src/poly.rs

```diff
@@ -96,7 +96,7 @@ fn get_test_vec(first: usize, second: usize) -> Vec<Fr> {
 }
 
 #[test]
-fn poly_test_div() {
+fn poly_test_long_division() {
```

Comment:
We should device whether we want "descriptive" name, or names that match C-kzg tests.

Reply:
Oh ok, sorry it is internal tests, all good.